### ISSUE TITLE
[Emergency] Uncaught ArgumentCountError: array_merge() does not accept unknown named parameters

### ISSUE THREAD

**Issue by baukezwaan** [silverleague/silverstripe-ideannotator#141]
There seems to be an issue on PHP8 in the getTags() function:

```
    public function getTags()
    {
        return (array)call_user_func_array('array_merge', $this->tags);
    }
```

If I understand correctly, the error is related a new feature of PHP 8.0 called '[Named Parameters]. It looks like this is solved by stripping the top-level array with `array_values`:

```
    public function getTags()
    {
        return (array)call_user_func_array('array_merge', array_values($this->tags));
    }
```

I'm happy to make a PR if this is the correct way of solving it.

**Comment by Firesphere** [comment]
Do you have an example of what syntax exactly causes the problem?

I've been away from PHP a bit too long to know the actual difference/change between PHP7 and 8 ;)

**Comment by adiwidjaja** [comment]
I just created a pull request for this issue (to be honest, I didn't see this discussion, I just found it).

**Comment by Firesphere** [comment]
All good, bugfixes don't need a whole discussion anyway :)

Resolved in #142
